game/ato/flightwaypointtype.py

```diff
@@ -43,8 +43,10 @@ class FlightWaypointType(IntEnum):
     DIVERT = 23
     INGRESS_OCA_RUNWAY = 24
     INGRESS_OCA_AIRCRAFT = 25
-    PICKUP = 26
-    DROP_OFF = 27
+    PICKUP_ZONE = 26  # Pickup Zone for cargo or troops
+    DROPOFF_ZONE = 27  # Dropoff Zone for cargo or troops
     BULLSEYE = 28
-    REFUEL = 29  # Should look for nearby tanker to refuel from.
-    STOPOVER = 30  # Stopover landing point using the LandingReFuAr waypoint type
+    AIR_REFUEL = 29  # Should look for nearby tanker to refuel from.
+    LAND_REFUEL = 30  # Stopover landing point using the LandingReFuAr waypoint type
+    AIR_DROP = 31  # Waypoint to drop paylod from the air
+    INGRESS_AIR_ASSAULT = 32
```

Comment:
I'd prefer it if these remained stable (the numbers and the use; we can change the names however we want as long as we don't change the meaning). It helps with save compat and costs us nothing.

```suggestion
    AIRPORT_PICKUP = 26  # Cargo pickup executed by an airport landing.
    AIRPORT_DROP_OFF = 27  # Cargo drop off executed by an airport landing.
    BULLSEYE = 28
    REFUEL = 29  # Should look for nearby tanker to refuel from.
    INGRESS_AIR_ASSAULT = 30
    CRATE_PICKUP = 31  # CTLD crate pickup location.
    CRATE_DROP_OFF = 32  # CTLD crate drop location.
```

afaict we don't actually need `AIR_DROP` or `LAND_REFUEL`.

`CRATE_PICKUP` is a more accurate description than `PICKUP_ZONE`, right? It's not actually a zone, it's the exact location of the crate, isn't it?

Reply:
It remained stable with the numbers IIRC. At least thats what i tried to achieve.

And regarding dropoff and pickup: they are zones actually in regards to ctld. The waypoint is the center of it. Crates are placed somewhere in that zone IIRC